integration-test/concat/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mediachain/concat-dependencies
+
+# Clone concat repo
+RUN mkdir -p /go/src/github.com/mediachain
+RUN rm -rf /go/src/github.com/mediachain/concat
+RUN git clone https://github.com/mediachain/concat /go/src/github.com/mediachain/concat
+WORKDIR /go/src/github.com/mediachain/concat
+ARG concat_ref=master
+RUN git checkout ${concat_ref}
+
+# install gx dependencies
+RUN gx install --global
```

Comment:
hmm, yeah, we can remove it; the build failure will remind us to update the base image if we forget 😄